test/e2e/minimatch/match_test.go

```diff
@@ -34,20 +34,24 @@ func TestFetchMatches(t *testing.T) {
 	be := pb.NewBackendClient(tc.MustGRPC())
 
 	var tt = []struct {
-		req  *pb.FetchMatchesRequest
-		resp *pb.FetchMatchesResponse
-		code codes.Code
+		description string
+		req         *pb.FetchMatchesRequest
```

Comment:
I would not have req / resp here - I understand this is not specific to this change - so you can refactor this in future. Preferably, you want the test structure to have actual test data - so in this case, I would say - gotProfile, gotMMF, wantMatches - is your test data. Then you create the req based on the data in the execution loop and compare matches, code in response to the wantCode. (Also, please stick to got / want naming - becomes much easier to read the test struct to see whats test data and whats expected response)